sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/HealthcareEntity.java

```diff
@@ -83,6 +79,15 @@ public String getText() {
         return this.text;
     }
 
+    /**
+     * Get the normalized text property: Healthcare normalized entity text as appears in the request.
```

Comment:
Change to : Get the healthcare normalized entity text as appears in the request.

Reply:
this doesn't represent the text as in the request. that is `Entity.Text`
The normalizedText is `Preferred name for the entity. Example: 'histologically' would have a 'name' of 'histologic'`